.github/CODEOWNERS

```diff
@@ -7,6 +7,7 @@
 # global owners are only requested if there isn't a more specific
 # codeowner specified below. For this reason, the global codeowners
 # are often repeated in package-level definitions.
-*      @ebuchman @cmwaters @tychoish @williambanfield @creachadair @thanethomson @sergio-mena @jmalicevic
+*       @ebuchman @cmwaters @tychoish @williambanfield @sergio-mena @jmalicevic @thanethomson @samricotta
 
-/spec @cason @sergio-mena @jmalicevic @milosevic
+# Spec related changes can be approved by the protocol design team
+/spec   @josef-widder @milosevic @cason @sergio-mena @jmalicevic
```

Comment:
it's potentially a bit confusing to have duplication between this and the full team.

Reply:
How so?